spec/unit/shoulda/matchers/active_model/numericality_matchers/odd_number_matcher_spec.rb

```diff
@@ -14,6 +14,12 @@
     it { expect(subject.diff_to_compare).to eq 2 }
   end
 
+  describe '#strict' do
+    it 'allows strict' do
```

Comment:
These test names can only be so descriptive, but maybe `it 'accepts a model that validates strictly'`?